search-relevancy-test/src/search_relevancy_test/boost_test.clj

```diff
@@ -0,0 +1,82 @@
+(ns search-relevancy-test.boost-test
+ "Functions to run the boost tests"
+ (:require
+  [camel-snake-kebab.core :as csk]
+  [search-relevancy-test.core :as core]
+  [search-relevancy-test.relevancy-test :as relevancy-test]
+  [search-relevancy-test.reporter :as reporter]))
+
+(def min-boost-value
+  "Default boost value to start at when running tests"
+  1.0)
+
+(def max-boost-value
+  "Default boost value to end at when running tests"
+  4.0)
+
+(def boost-increment
+  "How much to increment the boost for each test"
+  0.1)
+
+(defn- create-search-params
+  "Create additional search params to configure the field and boost in the s
+  search request"
+  [field value]
+  (format "&boosts[%s]=%.2f&boosts[include_defaults]=true" field value))
+
+(defn- create-test-args
+  "Create logging args for test. Want to set log description so in local_test_runs.csv
+  we can see what the field was set to when we compare results."
+  [field value]
+  ["-log-run-description"
+   (format "%s %.2f" field value)
+   "-log-history"
+   "false"])
+
+(defn- run-boost-test
+  "Run the relevancy tests with the boost for the given field configured through
+  search params. Return the result summary with the boost value assoc'd so we can
+  reference it later"
+  [field boost]
+  (let [params (create-search-params field boost)] ; append to search request
+    (println "***********************************************")
```

Comment:
You're using this string separator in other places -- I'd suggest a `def` at the top of the file (or, if used in multiple namespaces, with other common constants).